### PR TITLE
chore(cli): default svc logs limit to max when time filter is set

### PR DESCRIPTION
**Pull request by iamhopaul123**
Currently the default value for limit is always 10. However, this doesn't make sense when users set any time filtering flags. This PR addresses this issue.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by huanjani** [comment]
Can you show manual output, please?

**Comment by iamhopaul123** [comment]
> Can you show manual output, please?

Absolutely. Sorry I forgot to post the manual test.
```$ copilot svc logs --since 10m --env prod```
```
Only found one deployed service my-app in environment prod
copilot/my-app/8d50abf005 10.0.1.210 - - [24/Sep/2020 21:22:40] "GET / HTTP/1.1" 200 -
copilot/my-app/8d50abf005 10.0.0.120 - - [24/Sep/2020 21:22:49] "GET / HTTP/1.1" 200 -
copilot/my-app/8d50abf005 10.0.1.210 - - [24/Sep/2020 21:22:50] "GET / HTTP/1.1" 200 -
copilot/my-app/8d50abf005 10.0.0.120 - - [24/Sep/2020 21:22:59] "GET / HTTP/1.1" 200 -
copilot/my-app/8d50abf005 10.0.1.210 - - [24/Sep/2020 21:23:00] "GET / HTTP/1.1" 200 -
copilot/my-app/8d50abf005 10.0.0.120 - - [24/Sep/2020 21:23:09] "GET / HTTP/1.1" 200 -
copilot/my-app/8d50abf005 10.0.1.210 - - [24/Sep/2020 21:23:10] "GET / HTTP/1.1" 200 -
...
copilot/my-app/8d50abf005 10.0.1.210 - - [24/Sep/2020 21:32:10] "GET / HTTP/1.1" 200 -
copilot/my-app/8d50abf005 10.0.0.120 - - [24/Sep/2020 21:32:20] "GET / HTTP/1.1" 200 -
copilot/my-app/8d50abf005 10.0.1.210 - - [24/Sep/2020 21:32:20] "GET / HTTP/1.1" 200 -
copilot/my-app/8d50abf005 10.0.0.120 - - [24/Sep/2020 21:32:30] "GET / HTTP/1.1" 200 -
copilot/my-app/8d50abf005 10.0.1.210 - - [24/Sep/2020 21:32:30] "GET / HTTP/1.1" 200 -
```

**Review by huanjani — APPROVED** [review]

File: internal/pkg/ecslogging/service.go [hunk 1]
```diff
@@ -35,14 +37,26 @@ type ServiceClient struct {
...
 	TaskIDs   []string
 	// OnEvents is a handler that's invoked when logs are retrieved from the service.
 	OnEvents func(w io.Writer, logs []HumanJSONStringer) error
 }
 
+func (o WriteLogEventsOpts) limit() *int64 {
+	if o.Limit != nil {
+		return o.Limit
+	}
+	if o.StartTime != nil || o.EndTime != nil {
```
Comment (bvtujo) [inline internal/pkg/ecslogging/service.go:52]: 👍 Is there documentation of the default limits in our flag descriptions or usage help?

**Review by bvtujo — APPROVED** [review]

File: internal/pkg/ecslogging/service.go (hunk 1 above)
Comment (iamhopaul123) [reply internal/pkg/ecslogging/service.go:52]: great catch! since we now remove the default value for the flag, i think we should add to the description.

**Pull request merged**